tasks/main.yml

```diff
@@ -4,6 +4,8 @@
 ### Main entrypoint
 ###
 - include_tasks: create_cluster.yml
+  tags:
```

Comment:
The tests should not contaminate the codebase in such a way. The code used for testing can stay under the already-existing `./tests` dir.

Basically, there should be no trace of testing in the actual code of the role (same goes with any other app).

Reply:
Updated. Please review it again. :)